inference-engine/src/mkldnn_plugin/mkldnn_graph_optimizer.cpp

```diff
@@ -138,7 +138,8 @@ void MKLDNNGraphOptimizer::FuseConvolutionAndBias(MKLDNNGraph &graph) {
     auto& graphNodes = graph.GetNodes();
 
     auto isSutableParentNode = [](MKLDNNNodePtr node) {
-        return node->getType() == Convolution &&
+        return !node->isConstant() &&
```

Comment:
we need this check to avoid situations when we called isConstant() before real fusing and fuse dynamic node into const node, because in this case const type will be assigned for node but after fusing real type will be noConst
Also we inferred all const node on init graph stage and we won't get performance degradation